src/api/data/tree/importApi.ts

```diff
@@ -431,7 +433,7 @@ export class ImportApi {
     if (isJson) {
       this.logger.debug(`SObject Tree API results:  ${JSON.stringify(response, null, 4)}`);
 
-      if (response.hasErrors) {
+      if (response.hasErrors === true) {
```

Comment:
there's a union type on TreeResponse.  By throwing here, it narrows the type to one that has response.results (you'll see it have a TS error on `this.responseRefs = this.responseRefs.concat(response.results);` otherwise)